CMakeLists.txt

```diff
@@ -116,6 +124,10 @@ else()
 	endif()
 endif()
 
+if (GODOT_ENABLE_HOT_RELOAD)
+    set(GODOT_COMPILE_FLAGS "${GODOT_COMPILE_FLAGS} -D HOT_RELOAD_ENABLED")
```

Comment:
Other defines are added to the command-line without the space after `-D`, it'd be nice for that to be consistent:

```suggestion
    set(GODOT_COMPILE_FLAGS "${GODOT_COMPILE_FLAGS} -DHOT_RELOAD_ENABLED")
```